### ISSUE TITLE
Time to process a lemma has doubled in Coqc and Coqide

### ISSUE THREAD

**Issue by Zimmi48** [Zimmi48/bugzilla-test#4052]
Note: the issue was created automatically with bugzilla2github tool

# Bugzilla Bug ID: 4052
Date: 2015-02-17 14:26:41 +0100
From: @_charguer
To:   
CC:   coq-bugs-redist@lists.gforge.inria.fr, @_gares

Last updated: 2015-02-17 15:29:02 +0100

## Bugzilla Comment ID: 9458
Date: 2015-02-17 14:26:41 +0100
From: @_charguer

Posting this bug just in case you were not aware of this particular source of slowdown.

--------------- in Test.v --------------
Lemma test : True.
Proof. (* or Proof using. *)
  apply I.
Qed.
------------------------------

Executing: time coqc Test.v && time ~/softs/coq/bin/coqc Test.v

real	0m0.058s
user	0m0.031s
sys	0m0.027s

real	0m0.108s
user	0m0.077s
sys	0m0.032s
------------------------------

Arthur

## Bugzilla Comment ID: 9459
Date: 2015-02-17 14:38:16 +0100
From: @_gares

I'm not so sure this benchmark proves your point.
In general Coq got faster, and your message on coqdev confirms it.

In other words I suspect this micro-benchmark only shows that
there is some extra constant factor, not that the systems is
slower in general.  I mean, it could just be that the prelude
got bigger and it takes more time to load it.

## Bugzilla Comment ID: 9464
Date: 2015-02-17 15:29:02 +0100
From: @_charguer

Hum, it could be; I add the impression on other files that there is an overhead associated with entering a lemma. I'll see if I can reproduce it better. I close the bug in the meantime.
